pkg/manager/member/pd_scaler.go

```diff
@@ -127,7 +127,15 @@ func (psd *pdScaler) ScaleIn(meta metav1.Object, oldSet *apps.StatefulSet, newSe
 			return err
 		}
 		if leader.Name == memberName {
-			err = pdClient.TransferPDLeader(fmt.Sprintf("%s-pd-%d", tc.GetName(), 0))
+			var targetName string
+			lastOrdinal := helper.GetMaxPodOrdinal(*newSet.Spec.Replicas, newSet)
+			if ordinal == lastOrdinal {
+				targetName = PdPodName(tcName, helper.GetMinPodOrdinal(*newSet.Spec.Replicas, newSet))
+			} else {
+				targetName = PdPodName(tcName, lastOrdinal)
+			}
+
+			err = pdClient.TransferPDLeader(targetName)
```

Comment:
The logic from L124 should be as following：
```
if ordinal > minOrdinal {
  transfer leader to Pod with MinOrdinal
}
```
WDYT?

Reply:
In this case no need to `lastOrdinal := helper.GetMaxPodOrdinal(*newSet.Spec.Replicas, newSet)` this logic ?

Reply:
Yes, I think so, because this is the scaling in operation, we can just transfer the leader to the min ordinal Pod if it's not the min ordinal Pod and if it is, no need to transfer the PD leader, which is the same logic as it is.